src/fileio.cpp

```diff
@@ -0,0 +1,30 @@
+#include <cstring>
+#include <string>
+#include <iostream>
+#include <vector>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include "fileio.hpp"
+
+int FileIO::openFile(std::string fileName)
+{
+    int fd = open("./roms/Pokemon_Bleue.gb", O_RDONLY);
+    if (fd < 0) {
+        std::string error = strerror(errno);
+        std::cout << "error open : " << error << std::endl;
```

Comment:
Add //TODO comment to easily find this error so that when you'll make a better error handler you won't miss it.